website/docs/r/service_instance.html.markdown

```diff
@@ -49,3 +49,9 @@ An existing Service Instance can be imported using its guid, e.g.
 ```
 $ terraform import cf_service.redis a-guid
 ```
+
```

Comment:
Is this specific to the service_instance resource, or global timeouts that would apply on any resource ?

Reply:
It should be specific to the service_instance resource, and my recent commit enforces it: https://github.com/mevansam/terraform-provider-cf/pull/52/commits/72c9af8aa6d6e59fb8def1c342470e21e54d874c (See the StateChangeConf implementations)

Reply:
thanks